### PR TITLE
 COMMERCE-1633 show an error if a Country with that twoLettersISOcode already exists

### PR DESCRIPTION
**Pull request by luca-pellizzon**

### CODE REVIEW

**Comment by luca-pellizzon** [comment]
ci:test

**Comment by luca-pellizzon** [comment]
ci:test

**Comment by ethanbustad** [comment]
ci:test

**Comment by ethanbustad** [comment]
ci:test
Forgot to update private branch :'(

**Comment by luca-pellizzon** [comment]
ci:test

**Comment by luca-pellizzon** [comment]
ci:test

**Comment by luca-pellizzon** [comment]
ci:test

**Comment by luca-pellizzon** [comment]
ci:test

**Comment by ethanbustad** [comment]
ci:test

File: commerce-address-web/src/main/java/com/liferay/commerce/address/web/internal/portlet/action/EditCommerceCountryMVCActionCommand.java [hunk 1]
```diff
@@ -237,6 +239,12 @@ protected CommerceCountry updateCommerceCountry(ActionRequest actionRequest)
 		CommerceCountry commerceCountry = null;
 
 		if (commerceCountryId <= 0) {
+			if (_commerceCountryService.fetchCommerceCountry(
+					serviceContext.getCompanyId(), twoLettersISOCode) != null) {
+
+				throw new CommerceCountryAlreadyExistsException();
+			}
```
Comment (ethanbustad) [inline commerce-address-web/src/main/java/com/liferay/commerce/address/web/internal/portlet/action/EditCommerceCountryMVCActionCommand.java]: Shouldn't this validation be inside `commerceCountryLocalService.addCommerceCountry`? There's a unique index, so I think we'll always want to run this validation.

**Comment by luca-pellizzon** [comment]
ci:test

**Comment by luca-pellizzon** [comment]
@ethanbustad updates pushed

File: commerce-lang/src/main/resources/content/Language.properties [hunk 2]
```diff
@@ -487,6 +487,7 @@ the-sku-could-not-be-found=The SKU could not be found.
 the-specification-group-could-not-be-found=The specification group could not be found.
 the-subscription-entry-could-not-be-found=The subscription entry could not be found.
 the-tier-price-entry-could-not-be-found=The tier price entry could not be found.
+the-two-letter-iso-code-is-already-used=The two-letter iso code is already used.
```
Comment (ethanbustad) [inline commerce-lang/src/main/resources/content/Language.properties]: Shoot, I missed this last time -- `iso` should be capitalized in the value: `The two-letter ISO code is already used.` This is a pretty low priority fix tho.

Comment (luca-pellizzon) [reply commerce-lang/src/main/resources/content/Language.properties]: @ethanbustad labels updated

**Review by ethanbustad — APPROVED** [review]

**Comment by ethanbustad** [comment]
ci:test

**Comment by ethanbustad** [comment]
ci:test
Tests may pass now that https://github.com/liferay/com-liferay-commerce/pull/1436 is merged in.

**Pull request merged**